gsgmail_connector.py

```diff
@@ -793,7 +793,6 @@ def _handle_on_poll(self, param):
 
         # if user_email param is not present use login_email
         config = self.get_config()
-        self.debug_print("\n PRINTING CONFIG : {0} ".format(config))
         self.debug_print("\n PRINTING PARAM : {0} ".format(param))
```

Comment:
This kinda looks like developer debugging given that it is in all caps. This should also be printed anyway when invoking the action, right?

Reply:
I believe this would be printed when invoking, yes. 

It should not be printed, as the configuration contains sensitive information.